src/lib.js

```diff
@@ -19,6 +20,7 @@ export {Spinner, InlineSpinner} from './components/Spinner'
 export * from './components/Comment'
 export * from './components/CommentComposer'
 export * from './components/CommentTree'
+export * from './components/Overlay'
```

Comment:
General thing: I prefer explicit exports here. This file should give an overview of the whole api surface.

Also in this specific case, as far as i understand, you need to name the default – `export {default as Overlay}`.